package.json

```diff
@@ -15,12 +15,14 @@
     "ipfs-http-client": "^28.1.2",
     "jayson": "^2.0.6",
     "js-yaml": "^3.12.0",
-    "keytar": "^4.3.0",
     "node-fetch": "^2.3.0",
     "pkginfo": "^0.4.1",
     "prettier": "^1.13.5",
     "request": "^2.88.0"
   },
+  "optionalDependencies": {
```

Comment:
Guess what: I didn't even know optional dependencies were a thing in NPM/Yarn. 😳

Reply:
Yeah, they're kind of odd but pretty good for this sort of use case.